packages/suite/src/components/suite/NotificationCard.tsx

```diff
@@ -88,17 +93,17 @@ const Body = styled.div`
 const NotificationButton = styled(Button)<{
```

Comment:
This styled Button should not be there 🙏  My best guess is that the banner should render the [Warning](https://www.figma.com/file/Y2E8dcdNYrSMGIVRjYFXKp/%5Bs2.0%5D-Components?type=design&node-id=3172-19021&mode=dev) variant of the button, which hadn't been there at the time of me making the `Button` component, so it should be added first.